examples/usecases/multi-gpu-data-parallel-training.ipynb

```diff
@@ -0,0 +1,459 @@
+{
```

Comment:
<p>how do we know we speed up the training? can you do a comparison?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/855/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
<p>You can compare the training time.</p><p> </p><p>A benchmark of multi-gpu training is outside the scope for an example. It should be done as part of the implementation to test the efficiency of implementation. I added it to the RMP ticket a few weeks ago.</p>